src/cript/api/utils/save_helper.py

```diff
@@ -37,10 +101,13 @@ def get_uuid_from_error_message(error_message: str) -> str:
     UUID
         the UUID the API had trouble with
     """
+    bad_uuid = None
     if error_message.startswith("Bad uuid: "):
         bad_uuid = error_message[len("Bad uuid: ") : -len(" provided")].strip()
     if error_message.strip().startswith("Duplicate uuid:"):
         bad_uuid = error_message[len(" Duplicate uuid:") : -len("provided")].strip()
+    if bad_uuid is None or len(bad_uuid) != 36:
+        raise RuntimeError(f"The internal helper function `_get_uuid_from_error_message` has been called for an error message that is not yet implemented to be handled. error message {error_message}, found uuid {bad_uuid}.")
```

Comment:
this reminds me of the [Microsoft task manager story](https://youtu.be/Ve95Nh690l0?t=227) where the developer wrote in his phone number in the error message in case the user encountered